routes/home/index.js

```diff
@@ -18,8 +18,8 @@ const init = location.hash
       tests: [
         { code: '' },
         { code: 'data.find(x => x == 5000)' },
-        { code: 'data.find(x => x == 10000)' },
-      ],
+        { code: 'data.find(x => x == 10000)' }
+      ]
```

Comment:
Enable trailing commas to reduce diff maybe? You could add a prettier.rc I guess; no semis and trailing commas ;)